system/jlib/jstats.cpp

```diff
@@ -4011,6 +4051,277 @@ void verifyStatisticFunctions()
     }
 }
 
+class StatsAggregator : public CInterface
```

Comment:
What advantage does this class have over the CRuntimeSummaryStatisticCollection?